scvelo/plotting/scatter.py

```diff
@@ -1,10 +1,58 @@
 from .docs import doc_scatter, doc_params
-from .utils import *
 
 from inspect import signature
 import matplotlib.pyplot as pl
+from matplotlib.colors import is_color_like
 import numpy as np
 import pandas as pd
+from pandas import unique
+
+from anndata import AnnData
+from scvelo import settings
+import scvelo.logging as logg
+from scvelo.preprocessing.neighbors import get_connectivities
+from .utils import (
```

Comment:
would prefer using `__all__` in utils and star import instead. What do you think?